docs/contributing/accessibility/accessibility.mdx

```diff
@@ -0,0 +1,77 @@
+screen reader. This results in an audio read out of the page’s content, often with additional
+instruction depending on what semantic element is in focus.
+
+To understand more about screen reader and keyboard navigation, turn on `voice over` or download
+`NVDA` and try navigating your device with keyboard and screen reader.
+
+| WCAG Success Criteria                                                                                                                                                                                                                                                                                                                                                                                                         | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [1.1 Text alternatives](https://www.w3.org/WAI/WCAG21/Understanding/text-alternatives)                                                                                                                                                                                                                                                                                                                                        | Include `alt-text` or aria-labels for any non-text elements that conveys information not provided elsewhere. If elements are not informative, they may be marked as `decorative`.                                                                                                                                                                                                                                                                                                |
```

Comment:
> aria-labels
Should this be `aria-labels`?

Reply:
It’s not a usable attribute but a grouping of them so it seemed better to leave as non-code blocked

Reply:
That's fair.